src/components/Hero.tsx

```diff
@@ -14,8 +14,7 @@ export default function Hero() {
               <span className="block text-light-brown xl:inline">Pablo Felipe</span>
             </h1>
             <p className="mt-3 max-w-md mx-auto text-lg text-gray-500 sm:text-xl md:mt-5 md:max-w-3xl">
-              Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui lorem cupidatat commodo. Elit sunt amet
-              fugiat veniam occaecat fugiat aliqua.
+              I am a Computer Engineering student on the way to specialize in Web Development. Loving technology and programming since "...I've seen space it's been too long".
```

Comment:
When a website shows you are a student in the title, it will likely get you out of the best jobs. In my opinion, the fact you are a student it's not something really important to put on top.
```suggestion
              I am a Software Development that loves technology and programming since "...I've seen space it's been too long".
```